app/src/main/java/com/example/dungeonmapdigitizer/ui/home/HomeFragment.kt

```diff
@@ -29,8 +29,11 @@ class HomeFragment : Fragment() {
         val root: View = binding.root
 
         val textView: TextView = binding.textHome
-        homeViewModel.text.observe(viewLifecycleOwner) {
-            textView.text = it
+        homeViewModel.maps.observe(viewLifecycleOwner) {
+            maps -> val mapString = maps.joinToString{
```

Comment:
could we please have a line break after `maps ->`? I found this somewhat confusing to have a multi-line lambda body start on the same line as the argument and arrow part of the lambda. If this is typical for kotlin, I'll withdraw the request.